schemas/typescript/CameraCalibration.ts

```diff
@@ -1,4 +1,4 @@
-// Generated from CameraCalibration by @foxglove/schemas
+// CameraCalibration
```

Comment:
This comment is not useful.

Mentioning where the file was generated from and referencing the schema docs to discover how to leverage the schemas in Studio would be useful.

Reply:
removed for typescript